### PR TITLE
Python add library

### PR DESCRIPTION
**Pull request by maxbachmann**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by BastianZim** [comment]
Can you please make the CI output downloadable? Otherwise I cannot test the packages.

To do so, just add 
```yaml
azure:
  upload_packages: False
```
to conda-forge.yml in the root directory. 

Ref: https://conda-forge.org/docs/maintainer/conda_forge_yml.html?highlight=store_build_artifacts#configuring-conda-forge-yml

File: conda-forge.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ provider:
   linux_aarch64: default
   linux_ppc64le: default
   win: azure
+azure:
+  upload_packages: False
```
Comment (BastianZim) [inline conda-forge.yml:11]:
```suggestion
azure:
  store_build_artifacts: True
```
Sorry, I had originally posted the wrong code but then edited the comment but I guess you saw it before. This should do the trick.

Comment (maxbachmann) [reply conda-forge.yml]: I wrote `False` by mistake anyways, so it would have failed either way :)

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by maxbachmann** [comment]
no clue why the Python3.8 osx arm64 build crashes, since this did not happen without the uploading ...

**Comment by BastianZim** [comment]
Can confirm that the arm 3.10 build is working. Would you like me to test the others as well?

**Comment by BastianZim** [comment]
Might be related to this: https://github.com/conda-forge/conda-forge.github.io/issues/1823

**Comment by maxbachmann** [comment]
> Would you like me to test the others as well?

If it is not a lot of effort for you it, it probably would not hurt. If it is a lot of effort, it is probably not required, since they are build in the same way

**Comment by BastianZim** [comment]
No, not at all. Just one command. :)

**Comment by maxbachmann** [comment]
I restarted the Python3.8 osx arm64 build, which is now successful as well.

**Comment by BastianZim** [comment]
All tested and all working. Thanks for your work on this.

**Comment by BastianZim** [comment]
I have not had a proper look at the logs or build so apologies if this doesn't help but most cross-compilation errors that I've seen come from not passing in the conda-forge arguments `${CMAKE_ARGS}`.
I can see that you have done most of the stuff manually and you are just calling pip so this advice might not even apply but maybe this might be a starter?

Ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake

**Comment by maxbachmann** [comment]
Yes, I will release a version with these fixes later today, since those should be the most urgent ones. I doubt many people care about PyPy arch64/ppc64le.

**Comment by maxbachmann** [comment]
@ngam how can I tell conda-forge to add the build scripts for all possible platforms?
As far as I remember e.g. osx arm64 was added automatically by a bot for rapidfuzz/jarowinkler. However https://github.com/conda-forge/levenshtein-feedstock still does not build for most platforms. Is there anything I can do to add the builds there?

**Comment by BastianZim** [comment]
You can just add the name here for OSX: https://conda-forge.org/docs/maintainer/knowledge_base.html#apple-silicon-builds

And here for linux: https://conda-forge.org/docs/maintainer/knowledge_base.html#using-arch-rebuild-txt

**Pull request closed without merging**